modApiServer/test/org/aion/api/server/TxRecptLgTest.java

```diff
@@ -0,0 +1,162 @@
+        String contractB = "0x605060405234156100105760006000fd5b610015565b61011f806100246000396000f30060506040526000356c01000000000000000000000000900463ffffffff1680637fe7a5a014603157602b565b60006000fd5b3415603c5760006000fd5b608f60048080806010013590359091602001909192908035906010019091908080601001359035906000191690909160200190919290808060100135903590600019169090916020019091929050506091565b005b83839060001916908660008a8a7ff2433e32c57456caf7380aef3887a965ac2021deb7bf0989ee41a2cf4387b2f88989604051808383906000191690906000191690825281601001526020019250505060405180910390a45b505050505050505600a165627a7a723058209ec9a5d885689d4f729e93521792f26ebb06e8bc0d4b245489b602ddb289a1000029";
+
+        StandaloneBlockchain.Bundle bundle = (new StandaloneBlockchain.Builder())
+                .withValidatorConfiguration("simple")
+                .withDefaultAccounts()
+                .build();
+        StandaloneBlockchain bc = bundle.bc;
+        ECKey deployerAccount = bundle.privateKeys.get(0);
+
+        //======================
+        // DEPLOY contract A & B
+        //======================
+        BigInteger nonce = BigInteger.ZERO;
+        AionTransaction tx1 = new AionTransaction(
+                nonce.toByteArray(),
+                null,
+                new byte[0],
+                ByteUtil.hexStringToBytes(contractA),
+                1_000_000L,
+                1L
+        );
+        tx1.sign(deployerAccount);
+
+        nonce = nonce.add(BigInteger.ONE);
+        AionTransaction tx2 = new AionTransaction(
+                nonce.toByteArray(),
+                null,
+                new byte[0],
+                ByteUtil.hexStringToBytes(contractB),
+                1_000_000L,
+                1L
+        );
+        tx2.sign(deployerAccount);
+
+        BlockContext context = bc.createNewBlockContext(bc.getBestBlock(), List.of(tx1, tx2), false);
+        ImportResult result = bc.tryToConnect(context.block);
+        assertEquals(result, ImportResult.IMPORTED_BEST);
+
+        Address addressA = tx1.getContractAddress();
+        System.out.println("contract A = " + addressA);
+        Address addressB = tx2.getContractAddress();
+        System.out.println("contract B = " + addressB);
+        Thread.sleep(1000);
+
+        //======================
+        // CALL function A.AA
+        //======================
+        nonce = nonce.add(BigInteger.ONE);
+        AionTransaction tx3 = new AionTransaction(
+                nonce.toByteArray(),
+                addressA,
+                new byte[0],
+                ByteUtil.merge(ByteUtil.hexStringToBytes("0x9dacb3a3"), addressB.toBytes(), new DataWord(80_000).getData()),
```

Comment:
in order for people to more easily understand what the test is doing, it'd be nice if 0x9dacb3a3 and 80_000 were either

1) put into a well-named constant that describes what it means / how it was gotten
2) put some comment explaining what the values are / how it was gotten

Reply:
+1. I don't know what these values are, and I would like to :')

Reply:
correct me if im wrong but i think 0x9dacb3a3 could be the function hash of AA.

Reply:
@aionWilliam you are right.

Reply:
What's the correct way to retrive this function hash instead of hard coding it?